marinerescue-frontend/src/pages/HomePage.js

```diff
@@ -1,14 +1,137 @@
+                    </Link>
+                    <Link to="/leaderboard">Leaderboard</Link>
+                    <Link to="/about">About</Link>
+                    <Link to="/parents">Parents</Link>
+                    <a href="#playnow"><button id="nav-bar-button">Play Now</button></a>
+                    <a href="profile">Profile</a>
+                    {/* <div class="navbar-logo">
+                        <img src="images/profile-logo.png"></img>
+                    </div> */}
+                </div>
+
+                <div class="section">
+                    <div class="row">
+                        <div class="column">
+                            <h1>Join the Adventure</h1>
+                            <h2>Join the Marine Rescue team in <br />
+                                                        learning about how marine debris <br />
+                                                        affects ocean life and what you can do <br />
+                                                        to help in this citizen science journey!</h2>
+                            <button>Play Now</button>
+                        </div>
+                        <div class="column">
+                            <img id="strawberry-welcome" src="images/section-1-strawberry-welcome.png"></img>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="section">
+                    <div class="section-2">
+                        <h1>Your Journey So Far</h1>
+                        <h2>Earn points and badges while helping the Marine Rescue team learn more about their environment!</h2>
+                        <div class="row">
+                            <div class="column">
+                                <img id="icon" src="images/strawberry-and-friends-icon.png"></img>
+                                <h2>Strawberry and Friends</h2>
+                            </div>
+                            <div class="column">
+                                <img id="icon" src="images/marine-debris-icon.png"></img>
+                                <h2>Marine Debris</h2>
+                            </div>
+                            <div class="column">
+                                <img id="icon" src="images/beach-safety-icon.png"></img>
+                                <h2>Beach Safety</h2>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="section">
+                    <div class="section-3">
+                        <h1>Meet the Marine Rescue Team</h1>
+                        <h2>Get to know each marine animal's unique personality and story!</h2>
+                        <div class="row">
+                            <div class="column">
+                                <div class="content">
+                                    <img id="icon" src="images/strawberry.png"></img>
```

Comment:
invalid HTML to have all images have the same id. Do you mean this to be a class? Also, Img can be self-closing tag. 
```suggestion
                                    <img class="icon" src="images/strawberry.png" alt="Image of Strawberry the seal" />
```